test-suite/squidconf/time_units

```diff
@@ -15,17 +15,16 @@ dns_timeout 1 millisecond
 dns_timeout 0.001 second
 
 # maximum checks
-dns_timeout 9223372036854 milliseconds
-dns_timeout 9223372036 seconds
-dns_timeout 153722867 minutes
-dns_timeout 2562047 hours
-dns_timeout 106751 days
-dns_timeout 15250 weeks
-dns_timeout 7625 fortnights
-dns_timeout 3558 months
-dns_timeout 292.27 years
-dns_timeout 29.227 decades
-
+dns_timeout 2147483647854 milliseconds
+dns_timeout 2147483647 seconds
+dns_timeout 2562047 minutes
+dns_timeout 42700 hours
+dns_timeout 1779 days
+dns_timeout 254 weeks
+dns_timeout 127 fortnights
+dns_timeout 63 months
+dns_timeout 5.27 years
+dns_timeout 0.727 decades
```

Comment:
How were these new maximum values selected? I am asking because, AFAICT, in the official code, all values represent the same amount of time, but, in the PR code, only the first two lines match. For example,

* seconds/minutes lines in the official code: `9223372036/153722867 = 60.0000'
* seconds/minutes lines in the proposed code: `2147483647/2562047 = 838.1905'

If you just decreased the values so that the tests work on 32-bit platforms, please adjust them to actually represent the maximum on those platforms. Ideally, these values should be generated at ./configure time, but I do _not_ insist on that improvement in this PR.

Reply:
I chose 2^31 - some arbitrary amount, then rounded down. Using 2^31-1 also works for me, I'm not attached to these

Reply:
For these specific "maximum" tests to make any practical sense, they have to use the exact _maximum_ values. I understand that a quick fix requires using 32-bit maximums (which defeats the purpose of the test for 64-bit platforms), but we cannot have any delta beyond that.

In other words, all these tests must fail on a 32-bit platform if you increase the number by 1.

Reply:
Ok, I'll update the PR accordingly